packages/server/tests/acceptance/erc20.spec.ts

```diff
@@ -204,12 +206,16 @@ describe('@erc20 Acceptance Tests', async function () {
                                     await new Promise(r => setTimeout(r, 5000));
                                 });
 
+                                // issue #5652, HIP-584 mirror node does not support approve precompiles yet.
                                 it('emits an approval event', async function () {
-                                    const allowance = await contract.allowance(tokenOwner, spender);
-                                    const approvalEvent = (await tx.wait()).events.filter(e => e.event === Constants.HTS_CONTRACT_EVENTS.Approval)[0].args;
-                                    expect(approvalEvent.owner).to.eq(tokenOwnerWallet.address);
-                                    expect(approvalEvent.spender).to.eq(spenderWallet.address);
-                                    expect(approvalEvent.value).to.eq(allowance);
+                                    // still cover erc20 approve event
+                                    if(contract.address !== htsToken.address) {
```

Comment:
Not a fan of the fact the test file forces this kind of logic.
If we have to do this i'd prefer we do what the `'Relay can execute "eth_getCode" for ERC20 contract with evmAddress'` does and do it based off of the test title. The contract address isn't a clear relationship to me

Reply:
Done.